packages/expo/src/sideEffects.ts

```diff
@@ -0,0 +1,15 @@
+import './environment/validate';
+import './environment/logging';
+
+// load expo-asset immediately to set a custom `source` transformer in React Native
+import 'expo-asset';
+
+import Constants from 'expo-constants';
+
+import { installWebGeolocationPolyfill } from 'expo-location';
+
+if (typeof Constants.manifest.env === 'object') {
+  Object.assign(process.env, Constants.manifest.env);
```

Comment:
Is the issue with this on web that `process` is not defined? If so, we could do something like:
```
if (typeof process !== 'undefined') {
  Object.assign(...);
}
```